assets/messages.yml

```
@@ -351,6 +351,20 @@ messages:
         This is known to be an issue involving detection of the Java runtime.
         Please take a look at the Moderator’s Note on that ticket to see solutions to your issue.
 
+        %quick_links%
+      fillname: []
+  duplicate-of-mc-108:
+    - project: mc
+      name: Duplicate of MC-108
+      shortcut: quasi
+      message: |-
+       *Thank you for your report!*
```

Looks like the indentation here is not correct.
```suggestion
        *Thank you for your report!*
```